app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -97,13 +97,6 @@ $large-input-size: 50px;
   }
 }
 
-@include govuk-compatibility(govuk_template) {
```

Comment:
I can see the following apps with this:

- smart answers (but only in the non-public facing 'visualise' section, will check)
- datagovuk_find (will check)

Reply:
DGU works fine with this change, but errors horribly if `$govuk-use-legacy-palette` is removed, so this will definitely be a problem with version 5.

Reply:
Smart answers also works fine. It might be worth looking into why it's using legacy-palette as it might be possible to just remove it. Will address separately.